proto/penumbra/penumbra/core/transaction/v1alpha1/transaction.proto

```diff
@@ -201,10 +200,15 @@ message ActionView {
   }
 }
 
+// An authorization hash for a Penumbra transaction.
+message EffectHash {
+  bytes inner = 1;
+}
```

Comment:
Side note: I think we might want to split the effect hash code into its own crate, rather than putting it in the transaction crate. The reason is that all of the components need to be able to define their effecting data, but the overarching `transaction` crate depends on all the components.